airbyte-integrations/connectors/source-amazon-ads/unit_tests/test_report_streams.py

```diff
@@ -612,3 +609,37 @@ def test_streams_state_filter(mocker, config, state_filter, stream_class):
         assert params["stateFilter"] == ",".join(state_filter)
     else:
         assert state_filter is None
+
+
+@responses.activate
+def test_display_report_stream_single_record_type(config_gen):
```

Comment:
1. Both tests are the same except for the `custom_record_types` parameter. Deduplicate code by using `@pytest.mark.parametrize`. See `test_streams_state_filter` for an example.

2. Add two more test cases: `custom_record_types = None` and `custom_record_types = ["invalid_record_type"]` to make the test more exhaustive.

Reply:
sure, will do